scanner/cmd/scanner/main.go

```diff
@@ -103,19 +112,34 @@ func createGRPCService(backends *Backends) (grpc.API, error) {
 	if err != nil {
 		return nil, fmt.Errorf("identity extractor: %w", err)
 	}
-	// Create gRPC API service.
+	// Create gRPC API service and debug routes.
+	var customRoutes []routes.CustomRoute
+	for path, handler := range routes.DebugRoutes {
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         path,
+			Authorizer:    allow.Anonymous(),
+			ServerHandler: handler,
+			Compression:   true,
+		})
+	}
 	grpcSrv := grpc.NewAPI(grpc.Config{
-		CustomRoutes:       nil,
+		CustomRoutes:       customRoutes,
 		IdentityExtractors: []authn.IdentityExtractor{identityExtractor},
-		GRPCMetrics:        metrics.GRPCSingleton(),
-		HTTPMetrics:        metrics.HTTPSingleton(),
+		GRPCMetrics:        grpcmetrics.GRPCSingleton(),
+		HTTPMetrics:        grpcmetrics.HTTPSingleton(),
 		Endpoints: []*grpc.EndpointConfig{
 			{
 				ListenEndpoint: ":8443",
 				TLS:            verifier.NonCA{},
 				ServeGRPC:      true,
 				ServeHTTP:      false,
 			},
+			{
+				ListenEndpoint: ":9095",
```

Comment:
where did this port come from?

Reply:
The default debug port in other services.

Reply:
9095? prometheus is 9090 and pprof is 6060 typically iirc

Reply:
we're going to make it configurable later one anyway, so we can change it there if we want